code/game/objects/items/storage/secure.dm

```diff
@@ -47,25 +47,24 @@
 				return
 
 			if (isscrewdriver(W))
-				if (do_after(user, 20, TRUE, 5, BUSY_ICON_BUILD))
```

Comment:
Is the removal of the icon intended here?

Reply:
same.